lib/ignored-paths.js

```diff
@@ -142,25 +91,26 @@ function IgnoredPaths(options) {
      * @returns {array} raw ignore rules
      */
     function addIgnoreFile(ig, filepath) {
-        return ig.addIgnoreFile(filepath);
+        ig.ignoreFiles.push(filepath);
+        return ig.add(fs.readFileSync(filepath, "utf8"));
```

Comment:
Alright, so this is the in-my-poor-opinion thing :smile: that I'm gonna spelunk into the source code of eslint recently.